models/jules/R/model2netcdf.JULES.R

```diff
@@ -32,19 +32,19 @@ model2netcdf.JULES <- function(outdir) {
                 row.names = TRUE,
                 quote = FALSE)
     
-    vars <- ncdf4.helpers::nc.get.variable.list(nc)
+    vars <- names(nc[['var']])
```

Comment:
```suggestion
    vars <- names(nc[["var"]])
```

Double-quotes for strings, please -- we've been lax about this in the past but are now trying to follow [tidyverse style](https://style.tidyverse.org/syntax.html#quotes)